source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/FactSetRenderer.java

```diff
@@ -46,14 +46,14 @@ public static FactSetRenderer getInstance()
         return s_instance;
     }
 
-    static TextView createTextView(Context context, CharSequence text, FactSetTextConfig textConfig, HostConfig hostConfig, long spacing, ContainerStyle containerStyle)
+    static TextView createTextView(Context context, RenderArgs renderArgs, CharSequence text, FactSetTextConfig textConfig, HostConfig hostConfig, long spacing, ContainerStyle containerStyle)
```

Comment:
same as previously, I think moving RenderArgs as the last parameter may make more sense consistency wise